etc/DependencyInstaller.sh

```diff
@@ -529,18 +554,34 @@ EOF
         version=$(awk -F= '/^VERSION_ID/{print $2}' /etc/os-release | sed 's/"//g')
         spdlogFolder="/usr/local/lib/cmake/spdlog/spdlogConfigVersion.cmake"
         export spdlogFolder
-        _installUbuntuPackages "${version}"
-        _installCommonDev
-        _installOrTools "ubuntu" "${version}" "amd64"
-        _installUbuntuCleanUp
+        if [[ "${option}" == "base" || "${option}" == "all" ]]; then
+            if [[ ! -z "${PREFIX}" ]]; then
+                echo "Error: cannot install base packages locally, you need priviledged access" >&2
+                exit 1
+            fi
```

Comment:
Let's move this to a function to avoid copy and paste. Please note that the suggestion above should be addressed.